Makefile.toml

```diff
@@ -98,16 +98,30 @@ command = "target/release/darwinia"
 env     = { "FLAGS" = "--chain pangolin-dev" }
 extend  = "common-run"
 
+# Build for runtime benchmarks.
 [tasks.build-bench]
 command = "cargo"
 env     = { "FEATURES" = "all-natives,runtime-benchmarks" }
 extend  = "common-build"
 
+# Build for try runtime.
 [tasks.build-trt]
 command = "cargo"
 env     = { "FEATURES" = "all-natives,try-runtime" }
 extend  = "common-build"
 
+# Build for testnets.
+[tasks.build-tns]
+command = "cargo"
+env     = { "FEATURES" = "pangolin-native,pangoro-native" }
+extend  = "common-build"
+
+# Build for mainnets.
+[tasks.build-mns]
+command = "cargo"
+env     = { "FEATURES" = "crab-native,darwinia-native" }
+extend  = "common-build"
```

Comment:
We might need a README at the root of this repo to display the basic operation of this repo.

Reply:
Not now